build-scripts/version.py

```diff
@@ -0,0 +1,149 @@
+"""Write src/version.h and VERSION.txt from git, if available
+
+All commands emulate the Makefile's "version" target.
```

Comment:
can't un-resolve the resolved thread, but this is still an issue. We do not emulate Makefile's "version" target anymore, so this is either tautologous or confusing.